pkg/virt-controller/watch/network_test.go

```diff
@@ -261,6 +262,83 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			[]v1.VirtualMachineInterfaceRequest{},
 		),
 	)
+
+	Context("handle dynamic interface requests", func() {
+		var clusterConfig *virtconfig.ClusterConfig
+
+		const vmiHasPodNetwork = true
+
+		BeforeEach(func() {
+			clusterConfig, _, _ = testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
+		})
+
+		DescribeTable("for a VM with no interfaces",
+			func(requests []v1.VirtualMachineInterfaceRequest, expectedInterfaces []v1.Interface, isPodNetworkOnVmi bool) {
```

Comment:
nit: The test cases (Entry()) would read better if `isPodNetworkOnVmi` was the first argument.

Reply:
The comment is not relevant anymore, the param was removed.